roles/privacyidea/tasks/setup_nginx.yml

```diff
@@ -0,0 +1,38 @@
+---
+# Remove default Nginx configuration to avoid conflict
+- name: Remove default and PrivacyIDEA Nginx site and enable configurations
+  file:
+    path: "{{ item }}"
+    state: absent
+  loop:
+    - /etc/nginx/sites-available/default
+    - /etc/nginx/sites-enabled/default
+  become: true
+  notify:
+    - restart nginx
+
+- name: Copy Nginx configuration
+  template:
+    src: nginx_privacyidea.conf.j2
+    dest: /etc/nginx/sites-available/privacyidea.conf
+  notify: restart nginx
+
+- name: Create symbolic link to enable PrivacyIDEA site
+  file:
+    src: /etc/nginx/sites-available/privacyidea.conf
+    dest: /etc/nginx/sites-enabled/privacyidea.conf
+    state: link
+  notify:
+    - restart nginx
```

Comment:
same here, are all restarts necessary?

Reply:
Nginx needs to reload its configuration to recognize the new site, @aimee-889 , maybe i should reload nginx so that nginx  re-read its configuration files, without fully restarting the service

Reply:
if a reload is enough i would preferer a reload yes